packages/runtime-common/index-query-engine.ts

```diff
@@ -462,6 +464,21 @@ export class IndexQueryEngine {
     return { prerenderedCards, scopedCssUrls: [...scopedCssUrls], meta };
   }
 
+  async fetchCardTypeSummary(realmURL: URL): Promise<CardTypeSummary[]> {
+    let results = (await this.query([
+      `SELECT value
+       FROM realm_meta r
+       WHERE`,
+      ...every([
+        ['r.key = ', param(RealmMetaKey.CardTypeSummary)],
+        ['r.realm_url =', param(realmURL.href)],
+      ]),
+      `ORDER BY indexed_at`,
+      `LIMIT 1`,
```

Comment:
you should not have to LIMIT 1, the fact that you do is hiding a problem in that you should be querying by primary key